CHANGELOG.md

```diff
@@ -5,7 +5,6 @@
 * Add Item screen : Incorrect aria label. Refs UIIN-1104.
 * Select and move holdings with items or items to another instance. Refs UIIN-1098.
 * Move items between holdings. Refs UIIN-1097.
-* Disable fields in Preceding/Succeeding section, when Instance is controlled by Source MARC. Refs UIIN-1128.
```

Comment:
Should it be removed?

Reply:
It's looks like it is duplicate, because UIIN-1128 is a part of previous 3.0.2 release. line 37

Reply:
Gotcha. Makes sense then.